docs/manual/developer/04_style_guide.md

```diff
@@ -1,5 +1,23 @@
 # ComplianceAsCode Style Guide
 
+## Labels
+* Label-less approach should be followed
+* Labels must have a clear and objective description
+* New labels should be created in lowercase
```

Comment:
The convention before is that title cased labels are used for the change log and lowercase for everything else. But this seems to make sense.